src/main/kotlin/uk/gov/justice/digital/hmpps/bodyscan/apiclient/BodyScanPrisonApiClient.kt

```diff
@@ -30,4 +43,23 @@ class BodyScanPrisonApiClient(@Qualifier("prisonApiWebClient") private val webCl
       .block()
       ?: emptyList()
   }
+
+  fun getSentenceDetails(prisonNumber: String): SentenceDetails =
+    webClient.get().uri("/api/offenders/$prisonNumber/sentences")
```

Comment:
It would be better to use https://api.prison.service.justice.gov.uk/swagger-ui/index.html#/bookings/getOffenderBookingByOffenderNo. 

Otherwise people will think we're using sentence info - `/api/offenders/$prisonNumber` is much more standard and used by many more clients

Reply:
sorted